datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -61,14 +61,6 @@ use datafusion_common::{DataFusionError, Result};
 use datafusion_expr::binary_rule::binary_operator_data_type;
 use datafusion_expr::{binary_rule::coerce_types, ColumnarValue, Operator};
 
-// TODO move to arrow_rs
```

Comment:
The fact I didn't have to `use` the copy in arrow threw me for a little

Then I realized it is included by the following:
 
```rust
use arrow::array::*;
```